.github/workflows/pytest-action.yml

```diff
@@ -0,0 +1,26 @@
+name: Pytest Execution
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.10
```

Comment:
I think it need to be something like 3.10.11

build is throwing the error:
```
  Error: The version '3.1' with architecture 'x64' was not found for Ubuntu 22.04.
  The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```

Reply:
yep. fixed it